backend/src/test/java/com/bikes/backend/repository/MongoUserRepositoryTest.java

```diff
@@ -0,0 +1,89 @@
+package com.bikes.backend.repository;
+
+import com.bikes.backend.model.MongoUser;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Nested;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.data.mongo.DataMongoTest;
+import org.springframework.test.annotation.DirtiesContext;
+
+import java.util.Optional;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+@DataMongoTest
+class MongoUserRepositoryTest {
```

Comment:
One could ask if there is a lot of value in testing auto generated methods, as you are only providing method signatures in you repo ;)

Reply:
👍